source/_posts/2017/2017-01-10-jak-psat-zajimave-inzeraty-pro-ajtaky.md

```diff
@@ -0,0 +1,195 @@
+
+Doporučuju si přečíst [3 minižíky od Otty](http://ottocopy.cz/ebook-zdarma) - sekci *O nás*.
+
+### Buďte upřímní
+
+Uvádějte jen technologie, které opravdu máte. Ne že je plánujte, chcete, kolega se je učí, ale opravdu je máte. Když vám na pohovor přijde člověk, který se těší na PHP 7, a vy hobudete teprve nasazovat (kdo ví kdy), začne mít pocit, že vaříte z vody.
+
+Text musí být upřímný, aby čtenář dostal co očekává. Tak vám bude věřit a přijde znovu.
+
+### Kamarád taky rád
+
+Když u vás pracuje někdo šikovný, zmiňte ho v inzerátu. Lidé půjdou spíše za kamarádem nebo  někým v komunitě známým, než jen do firmy.
+
+Sám si pamatuju tyhle dvojice: 
+
+- Filip Procházka - DámeJídlo, Rohlík 
+- Ondrej Mirtes - Slevomat
+- Patrik Votoček - Shipito
+- Vašek Purchart - Lavito
+
+
+### Řeknete si o feedback - dovnitř i ven
+
+1. Řekněte si o feedback v práci: **co tě nakonec dovedlo k nám místo ke konkurenci**?
+
+2. Dejte to přečíst někomu zvenční. Proč?
+
+    - Vaši zaměstnanci vás přirozeně vidí lépe než někdo, koho neživíte.
+    - Dají vám upřímější zpětnou vazbu, bo je nemůžete vyhodit, a spíš řeknou, co potřebují pro navázání vztahu.
+    
+Napište třem lidem z cílovky: „Mrkni prosím na náš nový inzerát. Které 2 věci se ti nejvíc líbí a které 2 bys zlepšil?“
+
+       
+### Svlékněte se
+
+Ukažte, jak to doopravdy děláte. Důkaz místo slibů.
+
+Děláte open-source? Ven s ním. Vím, že Skrz, Jobs, Slevomat, PeckaDesign nebo Shopsys mají balíčky na Githubu. U ani jednoho z nich jsem si ale nevšiml odkazu v inzerátu.
+
+Více informací u mi může dát jenom náhled do zdrojáků aplikace. Open-source je velmi podceňovaný hiring nástroj.  
+    
+### Odlište se
+
+Prodejte své odlišnosti. Proto k vám lidi jdou, ne protože jste stejně jako ostatní děláte v PHP, Symfony, Nette a Doctrine. Taková informace by tam měla být (nejlíp v rámci open-source), ale určitě ne na předních místech.
+
+### Příklady přitažlivých odlišností
+
+Znám firmy, které:
+
+- Každý měsíc mají půldenní školení - dobrovolné ana přání progrmaátorů! 
+    - Znám taky "každý měsíc máme půldenní školení". To je ale povinné a zadání volí ředitel nebo jiné oddělení. Není větší pain, nže být na školení Jak dělat marketingu na Instagramu, když se chci naučit škálovat výkon přes sharding serverů. 
+
+- Dávají každémju ročně 10 000 Kč na vzdělání a může si sám vybrat, jak je investuje.
+- Každý poslední pátek v měsící dávají programátorům možnost refaktorovat jakoukoliv část kódu chtějí.
+       
+
+## Na velikosti a věku nezáleží
+ 
+Pojmy jako junior a senior mi nedávají smysl. Proč? Představte si, že máme 2 lidi, hledající práci.
+
+A. Senior může být vývojář s 10letou praxí, zná 1 framework, který se učil sám a nechce zkoušet druhý, protože ten jeden mu stačí. 
+B. Junior může být 19letý borec před výškou, který říkají, že ještě teda žádný framework nenspali, tak asi úplně nebude dobrej; to že umí Elastic, Doctrine, Nette a začíná se Symfony nevidí jako výhodu (true story)
```

Comment:
> dobrej; to že umí Elastic, Doctrine, Nette a začíná se Symfony nevidí jako výhodu (true story)

dobrý. To, že umí Elastic, Doctrine, Nette a začíná se Symfony, nevidí jako výhodu (true story).